edx_django_utils/private_utils.py

```diff
@@ -35,7 +36,7 @@ def __init__(self):
 
     """
     declared_middleware = getattr(settings, 'MIDDLEWARE', None)
-    if declared_middleware is None:
+    if declared_middleware is None and django.VERSION[0] < 2:
```

Comment:
This code should run only less than 2 django version. I have added condition here.
```
>>> import django
>>> django.VERSION
(1, 11, 26, 'final', 0)
>>> django.VERSION[0]
1
```